allure-web/src/app.js

```diff
@@ -11,6 +11,9 @@ import i18next, { initTranslations } from './util/translation';
 
 Behaviors.behaviorsLookup = behaviors;
 
+//https://github.com/d3/d3-timer/pull/21
+window.requestAnimationFrame = window.requestAnimationFrame.bind(window);
```

Comment:
Duplication. 
If you will rebase this commit onto latest master, you will see this change already commited.